projects/fsastorefrontstyles/scss/overrides/header.scss

```diff
@@ -284,6 +284,8 @@ header {
         font-weight: var(--cx-font-weight-semi);
         padding: 0.5rem 0 0.5rem;
         @include media-breakpoint-down(md) {
+          position: absolute;
```

Comment:
Needed to reposition links in mobile menu, since there are now possible 4 instead of 2.